### PR TITLE
Updated material and supplemental material

### PR DESCRIPTION
**Pull request by shriaas2898**
In reference to issue #998 
Added slide links to material topics and added other links with description
Removed Agile manifesto's external link as it had the same content as that present in the presentation
Moved topics which were not part of presentation to supplemental material

### CODE REVIEW

File: agile-development/intro-to-agile.md [hunk 1]
```diff
@@ -54,14 +54,14 @@ The benefits of Agile software development includes:
...
-- [Manifesto for Agile Software Development](http://agilemanifesto.org/)
-- [Scrum](https://www.youtube.com/watch?v=WxiuE-1ujCM)
-- [eXtreme Programming (XP)](https://en.wikipedia.org/wiki/Extreme_programming)
-- [Feature Driven Delopment (FDD)](https://en.wikipedia.org/wiki/Feature-driven_development)
-- [User stories](https://www.mountaingoatsoftware.com/agile/user-stories)
-- [Time-boxing](https://www.agilealliance.org/glossary/iteration)
+- [Manifesto for Agile Software Development](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_0)
+- [Scrum](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_90)
+     - [Video tutorial for scrum explaination](https://www.youtube.com/watch?v=WxiuE-1ujCM)
+- [User stories]
```
Comment (alodahl) [inline agile-development/intro-to-agile.md]:
```suggestion
- [What is a user story?]
```

File: agile-development/intro-to-agile.md [hunk 2]
```diff
@@ -54,14 +54,14 @@ The benefits of Agile software development includes:
...
 
 ### Materials
-- [Manifesto for Agile Software Development](http://agilemanifesto.org/)
-- [Scrum](https://www.youtube.com/watch?v=WxiuE-1ujCM)
-- [eXtreme Programming (XP)](https://en.wikipedia.org/wiki/Extreme_programming)
-- [Feature Driven Delopment (FDD)](https://en.wikipedia.org/wiki/Feature-driven_development)
-- [User stories](https://www.mountaingoatsoftware.com/agile/user-stories)
-- [Time-boxing](https://www.agilealliance.org/glossary/iteration)
+- [Manifesto for Agile Software Development](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_0)
+- [Scrum](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_90)
```
Comment (alodahl) [inline agile-development/intro-to-agile.md]:
```suggestion
- [Techtonica Slides: Agile Project Management](https://docs.google.com/presentation/d/1SdCigo6K4aW8p-SZRA0vdI6w628pwhjzf-oX8UKIhS8/edit#slide=id.g1fcf0059a0_0_90)
```

File: agile-development/intro-to-agile.md [hunk 3]
```diff
@@ -54,14 +54,14 @@ The benefits of Agile software development includes:
...
+     - [Video tutorial for scrum explaination](https://www.youtube.com/watch?v=WxiuE-1ujCM)
```
Comment (alodahl) [inline agile-development/intro-to-agile.md]:
```suggestion
     - [Video: Explaining Scrum in less than 120 seconds](https://www.youtube.com/watch?v=WxiuE-1ujCM)
```

**Comment by shriaas2898** [comment]
Sure! Will make the required changes

**Comment by shriaas2898** [comment]
Made the changes in #1079 please review it.

**Comment by alodahl** [comment]
Does that mean this PR should close?

**Pull request closed without merging**